lti_redirect/templates/home.html

```diff
@@ -10,6 +10,12 @@
 <body>
 <div class="navbar">
     <a href="{% url 'home' %}" class="navbar-brand">LTI Redirect TOOL</a>
+    {% if user.is_authenticated and user.is_superuser %}
+    <a class="navbar-item" href="{% url 'admin:index' %}">Admin</a>
+    {% endif %}
+    {% if user.is_authenticated %}
+    <span class="navbar-user">{{ user.username }}</span>
+    {% endif %}
 </div>
```

Comment:
This set up add the user name to the view and enables admin url is the user is super user. You need to enable if the user is a super user or not by going to Database `auth_user`.